### PR TITLE
b2c-refactoring

### PR DESCRIPTION
**Pull request by gorav-sharma-metacube**
## PR Description
CC-16761: Refactor api_b2c environment variables file to match the new format

### CODE REVIEW

**Comment by gorav-sharma-metacube** [comment]
Please check attached log file
[results.zip](https://github.com/spryker/robotframework-suite-tests/files/8928258/results.zip)
.

File: resources/environments/environments_api_b2c.json [hunk 1]
```diff
@@ -9,267 +9,462 @@
...
+                "sku": "035",
+                "name": "Canon PowerShot N"
+            },
+
+            "with_options":{
+                "sku": "012",
+                "name": "Canon IXUS 165"
+            },
+
+            "multiple_variants":{
```
Comment (OKrivtsova) [inline resources/environments/environments_api_b2c.json:402]: we have products with multiple variants in 3 places in one file. here, in here
"abstract_product_with_variants":{
            "concretes_3": "127",
            "concretes_name_3": "HP Z 620",
            "concretes_superattribute_3": "total_storage_capacity"
        },

and in here
            "multivariant":{
            "sku_variant1": "130_29285281",
            "sku_variant2": "130_24326086"
            }
        },

please consolidate them

File: resources/environments/environments_api_b2c.json [hunk 2]
```diff
@@ -9,267 +9,462 @@
...
+            "with_stock_one_concrete_with_superattribute_name_superattribute": "form_factor",
+            "with_superattribute_sku": "156_32018944",
+            "with_superattribute_meta_keywords": "Acer,Communication Electronics",
+            "with_superattribute_meta_description": "Portable charm With a bevy of trendy, eye catching colours to suit any lifestyle and show your personality, this slim and lightweight tablet designed to be"
```
Comment (OKrivtsova) [inline resources/environments/environments_api_b2c.json:357]: we should subdivide here

in subsections:
with_stock_one_concrete_with_superattribute : name, description, sku

and so on for this section

Comment (gorav-sharma-metacube) [reply resources/environments/environments_api_b2c.json]: fixed

File: resources/environments/environments_api_b2c.json (hunk 1 above)
Comment (gorav-sharma-metacube) [reply resources/environments/environments_api_b2c.json]: fixed

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**